src/topology/algebra/ordered.lean

```diff
@@ -995,6 +995,107 @@ lemma cinfi_of_cinfi_of_monotone_of_continuous {f : α → β} {g : γ → α}
 by rw [infi, cInf_of_cInf_of_monotone_of_continuous Mf Cf
   (λ h, range_eq_empty.1 h ‹_›) H, ← range_comp]; refl
 
+lemma is_connected.forall_Icc_subset {s : set α} (hs : is_connected s)
```

Comment:
You are in a section with the standing assumption `conditionally_complete_linear_order α`. However, I guess this lemma should hold if you only assume you are on a (decidable?) linear order, right?

Reply:
Thank you! Moved.